frontend/src/components/organisms/form-sections/OrganisationSection/DisconnectionModalCard.js

```diff
@@ -0,0 +1,19 @@
+import ConfirmationModal from '../../ConfirmationModal';
+
+export const DisconnectionModalCard = ({
+  urlForDisconnectionPrompt,
+  setUrlForDisconnectionPrompt,
```

Comment:
the setUrlForDisconnectionPrompt props should not be necessary. Plz remove this and replace it with more readable props. (PP session might be usefull here)